### PR TITLE
Added observed holidays for Ukraine

### PR DESCRIPTION
**Pull request by KJhellico**
Added observed holidays for Ukraine and tests (last years and full for observed dates)

### CODE REVIEW

**Comment by KJhellico** [comment]
Uncovered code is for case when Easter is Apr 30, and nearest such day will be in 2062. Should I add test for it?

File: holidays/countries/ukraine.py [hunk 1]
```diff
@@ -12,15 +12,42 @@
...
+    SUN,
+    WEEKEND,
+)
 from holidays.holiday_base import HolidayBase
 
 
 class Ukraine(HolidayBase):
     """
+    Current holidays list:
     http://zakon1.rada.gov.ua/laws/show/322-08/paran454#n454
```
Comment (arkid15r) [inline holidays/countries/ukraine.py:41]:
```suggestion
    https://zakon1.rada.gov.ua/laws/show/322-08/paran454#n454
```

**Review by arkid15r — APPROVED** [review]
> Uncovered code is for case when Easter is Apr 30, and nearest such day will be in 2062. Should I add test for it?

The more coverage the better. Go for it!

This PR looks great, thanks for the effort!

**Comment by KJhellico** [comment]
Seems tests is OK

**Comment by dr-prodigy** [comment]
Thx :+1:

**Pull request merged**